docs/Documentation/How-To-Use.md

```diff
@@ -20,13 +20,14 @@ For example you have a `reports` table and want to save a PDF to it, you would t
 public function initialize(array $config)
 {
 	parent::initialize($config);
-	$this->table('reports');
+	
+	$this->setTable('reports');
 
-	$this->hasOne('PdfFiles', [
-		'className' => 'Burzum/FileStorage.PdfFiles',
+	$this->hasOne('Files', [
```

Comment:
It might be a good idea to stick to PdfFiles here for two reasons:

*  I don't know if it is still a problem, but Cake had the issue in the past, that unique assoc names made sense because we had the issue that the table registry screwed up AFAIR.
* If you have Pdf Files and Image files, you want a clear naming. You weakened the expression from specific to very generic.